tensorflow_addons/image/median_filter_2D.py

```diff
@@ -0,0 +1,86 @@
+import tensorflow as tf
+from tensorflow.python.ops.image_ops_impl import _Assert3DImage
```

Comment:
This is unlikely to be added as a public API. I would be in favor of copying the code into a util file of some sorts within tfa.image

Reply:
As an aside are there other image functions in addons that should utilize this?

Reply:
_Assert3DImage is not useful in the context. As i am explicitly checking for 3D tensor later. Removing it.